x-pack/filebeat/input/httpjson/input.go

```diff
@@ -229,13 +333,15 @@ func (in *httpjsonInput) processHTTPRequest(ctx context.Context, client *http.Cl
 				if in.config.Pagination.ExtraBodyContent != nil {
```

Comment:
The switch statement above assign the result of the type assertion to a variable to avoid needing the second `v.(string)`.